### PR TITLE
Para and Image are now of same size and added hoverable buttons in th…

### LINKED ISSUE #532: [Enhacement] Improve UX
**Description**
- Display information in a card to effectively utilize space. Also, reference links could be replaced with a button with a hover effect.
image
- In the used by section, display the company's logo on a marquee (logos moving from left to right)

---
**Additional Context**
**Note**: The above mentioned suggestions are open to discussion

### Contributor Resources

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- See the [Contributing to Meshery.io Website](https://github.com/layer5io/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- See Meshery site designs in this [Figma designfile](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=110%3A1). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by AnshumanDhiman**
…e links within the paragraphs w/signoff

Signed-off-by: AnshumanDhiman <67755381+AnshumanDhiman@users.noreply.github.com>

As discussed with lee I had removed the marquee and the color of slack icon is also set to default.
The paragraph and the supporting image will now occupy same space and instead of href links added hoverable buttons to them

This PR fixes #532 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -95,12 +96,12 @@ <h2>Use the <strong>Service Mesh Performance</strong> standard</h2>
 				<h2>What are <strong>configuration best practices</strong> and patterns?</h2>
 				<p> Assess your service mesh configuration against deployment and
 					operational best practices with Meshery's configuration validator.
-				<div align="center">
-					See the <a href="https://docs.meshery.io">service mesh patterns</a>
+				<div align="center" class="button adopter-button">
+					<a href="https://docs.meshery.io">See the service mesh patterns</a>
```
Comment (warunicorn19) [inline index.html:100]:
```suggestion
					<a href="https://docs.meshery.io/functionality/pattern-management">See the service mesh patterns</a>
```
Let's change the link to this.

**Review by warunicorn19 — CHANGES_REQUESTED** [review]
Could you please add the button to this too.

Screenshot 2021-08-31 at 2 00 17 AM

**Comment by leecalcote** [comment]
@AnshumanDhiman heads-up: it's always good to ensure that an issue is approved by a maintainer before implementing.

Feedback:

- The buttons are too close to their description
- There's missing whitespace between "(SMP)" and the text before it.
- "See the testing SMI conformance" is grammatically incorrect.
- Not all of the other sections have buttons added. Let's be consistent with either using buttons or not using them.

**Comment by leecalcote** [comment]
@AnshumanDhiman I think very well of your efforts here. Please keep it up.

**Comment by AnshumanDhiman** [comment]
Thanks @leecalcote every PR that I generate gives me more confidence and enthusiasm towards contributing to the layer5 community😀

**Comment by AnshumanDhiman** [comment]
> @AnshumanDhiman heads-up: it's always good to ensure that an issue is approved by a maintainer before implementing.
> 
> Feedback:
> 
> * The buttons are too close to their description
> * There's missing whitespace between "(SMP)" and the text before it.
> * "See the testing SMI conformance" is grammatically incorrect.
> * Not all of the other sections have buttons added. Let's be consistent with either using buttons or not using them.

What I think is that the buttons gives the visitor and the website a better way to interact with each other😅

**Comment by leecalcote** [comment]
> > @AnshumanDhiman heads-up: it's always good to ensure that an issue is approved by a maintainer before implementing.
> > Feedback:
> > 
> > * The buttons are too close to their description
> > * There's missing whitespace between "(SMP)" and the text before it.
> > * "See the testing SMI conformance" is grammatically incorrect.
> > * Not all of the other sections have buttons added. Let's be consistent with either using buttons or not using them.
> 
> What I think is that the buttons gives the visitor and the website a better way to interact with each other😅

I agree. The buttons are an improvement. 👍 We need to give them a little breathing room from the text above them, though.

**Comment by leecalcote** [comment]
...and you did just that. Nice!

We'll have to think of other appropriate links to have as buttons for the other sections. Thanks for this, @AnshumanDhiman 💯

**Review by leecalcote — APPROVED** [review]

**Pull request merged**